Tests/ZipManyTests.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  ZipManyTests.swift
+//  CombineExtTests
+//
+//  Created by Jasdev Singh on 16/03/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import XCTest
+import Combine
+import CombineExt
+
+final class ZipManyTests: XCTestCase {
+    private var subscription: AnyCancellable!
+
+    private enum ZipManyError: Error {
+        case anError
+    }
+
+    func testOneEmissionZipping() {
+        let first = PassthroughSubject<Int, Never>()
+        let second = PassthroughSubject<Int, Never>()
+        let third = PassthroughSubject<Int, Never>()
+
+        var results = [[Int]]()
+        var completed = false
+
+        subscription = first
+            .zip(with: second, third)
+            .sink(receiveCompletion: { _ in completed = true },
+                  receiveValue: { results.append($0) })
+
+        first.send(1)
+        second.send(2)
+        third.send(3)
+
+        XCTAssertEqual(results, [[1, 2, 3]])
+        XCTAssertFalse(completed)
+        first.send(completion: .finished)
+        XCTAssertTrue(completed)
+    }
+
+    func testMultipleEmissionZippingEndingWithAnError() {
```

Comment:
can we add one more tests one one of the publishers complete with `.finished` ?

Reply:
Sure thing! I’ll add it alongside a couple of other tidying-up commits. 👌🏽